src/test/java/org/reaktivity/nukleus/sse/internal/SseConfigurationTest.java

```diff
@@ -30,4 +32,10 @@ public void shouldVerifyConstants() throws Exception
     {
         assertEquals(SSE_INITIAL_COMMENT_ENABLED.name(), SSE_INITIAL_COMMENT_ENABLED_NAME);
     }
+
+    @Test
+    public void shouldMatchChallengeEventTypeConfigName()
+    {
+        assertEquals(EVENT_TYPE_NAME, CHALLENGE_EVENT_TYPE.name());
+    }
```

Comment:
Do we still need `EVENT_TYPE_NAME` constant separate from `CHALLENGE_EVENT_TYPE.name()` ?
If so, then please rename `EVENT_TYPE_NAME` to `CHALLENGE_EVENT_TYPE_NAME`, otherwise let's remove `EVENT_TYPE_NAME` and this unit test.